### PR TITLE
Update template

### PR DESCRIPTION
**Pull request by alax99**
Updating template

### CODE REVIEW

**Comment by zixuan75** [comment]
Very sorry, but we're limited to 1 review every 10 minutes.

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 ## How?
-GO fill in the missing parts.
+<!-- GO fill in the missing parts. -->
 1. Go to '...'
 2. Look at '...'
 3. Scroll down to '...'
 4. Fix '...'
 5. Done.
+
+We're actually very well done on some damn pull requests. Now not on amy of them anymore.
```
Comment (zixuan75) [inline PULL_REQUEST_TEMPLATE.md]: Why did you say "damn"? Also there is a misspelling on "any".
```suggestion
 We're actually very well done on some pull requests. Now not on any of them anymore.
```

Comment (zixuan75) [reply PULL_REQUEST_TEMPLATE.md]: Hello?

Comment (zixuan75) [reply PULL_REQUEST_TEMPLATE.md]: Out of date.

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 ## How?
-GO fill in the missing parts.
+<!-- GO fill in the missing parts. -->
 1. Go to '...'
 2. Look at '...'
 3. Scroll down to '...'
 4. Fix '...'
 5. Done.
+
+We're actually very well done on some pull requests. We're not on any of them anymore.
```
Comment (zixuan75) [inline PULL_REQUEST_TEMPLATE.md]: Now very good.

**Comment by zixuan75** [comment]
Alright.

File: PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (alax99) [reply PULL_REQUEST_TEMPLATE.md]: Huh

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # :rainbow: The React:laughing: Project:star::stars:: [![Coverage Status](https://coveralls.io/repos/github/zixuan75/react-project/badge.svg)](https://coveralls.io/github/zixuan75/react-project?branch=master):fire:
 
-Welcome to The React Project!!:2nd_place_medal:
+Welcome to The React Project!:2nd_place_medal:
```
Comment (zixuan75) [inline README.md]: Yeah, just one exclamation mark is enough. Just one.

**Comment by zixuan75** [comment]
DOne.

**Comment by alax99** [comment]
Uh ok.

**Pull request merged**